examples/Makefile.am

```diff
@@ -17,6 +17,7 @@ AM_CFLAGS = \
 	-I../tests \
```

Comment:
Can you do something like this:

extra_Includes = \
-I $(top_srcdir) \ #to compile within yocto

and then you add the extra_includes wherever you need.  

That way can help us identify in the future why this was added and maybe modify it.